.github/actions/pkgci-setup/action.yml

```diff
@@ -36,7 +36,7 @@ runs:
       uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
       with:
         path: .uv-cache
-        key: ${{ runner.os }}-uv-py${{ inputs.python-version }}-${{ hashFiles('requirements-iree-pinned.txt', 'pytorch-cpu-requirements.txt', 'sharktank/requirements.txt', 'sharktank/requirements-tests.txt', 'shortfin/requirements-tests.txt') }}
+        key: ${{ runner.os }}-uv-py${{ inputs.python-version }}-${{ hashFiles('requirements-iree-pinned.txt', 'pytorch-cpu-requirements.txt', 'sharktank/requirements.txt', 'sharktank/requirements-tests.txt', 'shortfin/requirements-tests.txt', 'sharktuner/requirements.txt') }}
```

Comment:
I took another look: This composite action is only used in `.github/workflows/pkgci_shark_ai.yml` and that workflow only tests shortfin and sharktank. Thus there is no need to install sharktuner deps. Please revert to the previous state.